extensions/adobe/experience/campaign/experienceevent.schema.json

```diff
@@ -146,7 +146,7 @@
           "type": "integer",
           "description": "The campaign activity originating this message."
         },
-        "https://ns.adobe.com/experience/campaign/campaign/id": {
+        "https://ns.adobe.com/experience/campaign/marketingCampaign/id": {
```

Comment:
This would work better as `https://ns.adobe.com/experience/campaign/marketingCampaignID`, see #277 and #284 for details.